include/ignition/gazebo/EventManager.hh

```diff
@@ -47,7 +47,7 @@ namespace ignition
     /// occur.
     ///
     /// See \ref ignition::gazebo::events for a complete list of events.
-    class IGNITION_GAZEBO_VISIBLE EventManager
+    class EventManager
```

Comment:
I guess this is related to https://github.com/ignitionrobotics/ign-gazebo/issues/545. I think we need a different fix, because Windows users will need to see and use this class.

The same goes for the other places where `IGNITION_GAZEBO_VISIBLE` is being removed.